evaluation/experiments.md

```diff
@@ -86,3 +86,35 @@ This starts the redis server running in the background, using `/mnt/pmem/` to st
 ```
 ./bin/ycsb load redis -s -P workloads/workloada -p redis.host=127.0.0.1 -p redis.port=6379
 ```
+
+# Windows
+
+## Setup
+1. Install JDK 22
```

Comment:
Could you add a link or more info on how to do this installation?

Reply:
updated.